### ISSUE TITLE
RoiPooling difference

### ISSUE THREAD

**Issue by fferroni** [yhenon/keras-frcnn#207]
Hello,

I noticed that in the theano backend for roi pooling you use 
pooled_val = K.max(xm, axis=(2, 3))
but in tensorflow, you are using tf.image.crop_and_resize which rescales _bilinearly_
so in tensorflow, you are not actually doing the same thing as in theano (and in the paper)

could you clarify?

**Comment by Abhijit-2592** [comment]
Even I have the same doubt

**Comment by mkocabas** [comment]
In recent papers like Mask RCNN[1] and Speed Accuracy Trade-offs for Modern Obeject Detectors[2], it is stated that using interpolation improves the performance significantly. Tensorflow crop and resize layer designed for that purpose.

[1]: https://arxiv.org/abs/1703.06870
[2]: https://arxiv.org/abs/1611.10012

On 1 Nov 2017, at 08:38, Abhijit Balaji <notifications@github.com<mailto:notifications@github.com>> wrote:


Even I have the same doubt

**Comment by Abhijit-2592** [comment]
@mkocabas  Thanks

**Comment by mrqorib** [comment]
@Abhijit-2592 Is the paper implementation of the RoI Pooling the same as the theano or is it the same as the tensorflow backend?

**Comment by Abhijit-2592** [comment]
@mrqorib it is same in theano background. But tf's implementation outperforms it
